### PR TITLE
Add assets image getter

### LINKED ISSUE #52: Create an ImageGetter to load image from assets

### PR DESCRIPTION
**Pull request by danielpassos**
This closes #52

### CODE REVIEW

**Comment by dschuermann** [comment]
How is this different from HtmlLocalImageGetter ?

**Comment by danielpassos** [comment]
I thought HtmlLocalImageGetter is looking for images only into res/drawable not into assets also. Am I wrong?

**Comment by dschuermann** [comment]
Hmm, yes HtmlLocalImageGetter only looks in res/drawable. Still not sure if we need a different ImageGetter for this.

Can we make this the fallback for HtmlLocalImageGetter when the drawable is not found in res/drawable?

**Comment by danielpassos** [comment]
Yes we can, but I don't agree with that. IMO classes need to be only one responsibility and if you are about to load imagens it's probably all in the same place (assets/raw/http). 

One responsibility per class also make the performance better because the classe don't need to test/find it in lot of different places.

Anyway It's your project, so, it's your call :)

**Comment by dschuermann** [comment]
Good argument, please put a license header on top of the file and I'll merge.

**Comment by danielpassos** [comment]
@dschuermann Sure. I'll also add this in info in the `README.md`

**Comment by danielpassos** [comment]
Done!

File: HtmlTextView/src/main/java/org/sufficientlysecure/htmltextview/HtmlAssetsImageGetter.java [hunk 1]
```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright (C) 2014-2016 Dominik Schürmann <dominik@dominikschuermann.de>
+ * Copyright (C) 2014 drawk
```
Comment (dschuermann) [inline HtmlTextView/src/main/java/org/sufficientlysecure/htmltextview/HtmlAssetsImageGetter.java]: remove these two lines and put your name in with 2016

Comment (danielpassos) [inline HtmlTextView/src/main/java/org/sufficientlysecure/htmltextview/HtmlAssetsImageGetter.java]: Done!

**Pull request merged**